chai-immutable.js

```diff
@@ -689,14 +689,14 @@
      * @api public
```

Comment:
Could you add the doc for these additions please?

Similarly to [this](https://github.com/chaijs/chai/blob/9a6610eb1e6b6da1645e9aee26ba452c227aa1dc/lib/chai/core/assertions.js#L2055), after `* @param {Collection} expected`, that would be `* @param {String} message _optional_`